docs/CloudPermissions.md

```diff
@@ -0,0 +1,78 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
```

Comment:
Also, this doc assumes that users will create an environment based on the manual guideline but it never mentions it, so it is very misleading. So the title and the descriptions should be updated.

Reply:
Updated the heading and sentence in 0aedc65